README.md

```diff
@@ -151,7 +151,7 @@ docker build -t irods-http-api-runner \
 If all goes well, you will have a containerized iRODS HTTP API server! You can verify this by checking the version information. Below is an example.
 ```bash
 $ docker run -it --rm irods-http-api-runner -v
-irods_http_api v0.1.0-f6f6411
+irods_http_api v0.2.0-f6f6411
```

Comment:
Hmm. That build SHA is kind of painful to deal with in the README.

Not sure how to keep it in sync with the actual repo yet.

Reply:
yes, this is not... obvious.

Reply:
could use <build_sha> here too... and maybe nobody will get confused...

Reply:
i tried it - see what you think.

Reply:
That works. If it confuses someone, we'll adjust it.

Reply:
excellent.